cli/sawtooth_cli/admin_command/genesis.py

```diff
@@ -54,6 +60,11 @@ def add_genesis_parser(subparsers, parent_parser):
         help='file or files containing batches to add to the resulting '
         'GenesisData')
 
+    parser.add_argument(
+        '--ignore-required-settings',
+        action='store_true',
+        help='skip checking for settings that are required at genesis')
```

Comment:
Make the same change(s) as on line 114 of main.rs.

Reply:
Updated